components/Onboarding/Section.js

```diff
@@ -0,0 +1,113 @@
+import React, { Fragment } from 'react'
+import { css } from 'glamor'
+
+import DoneIcon from 'react-icons/lib/md/done'
+
+import {
+  Button,
+  Loader,
+  mediaQueries,
+  colors
+} from '@project-r/styleguide'
+
+const HEADING_FONT_SIZE_MOBILE = 22
+const HEADING_FONT_SIZE_DESKTOP = 30
+
+const styles = {
+  p: css({
+    marginBottom: 20
+  }),
+  section: css({
+    cursor: 'pointer',
+    display: 'flex',
+    justifyContent: 'space-between',
+    alignItems: 'baseline',
+    borderTop: `1px solid ${colors.divider}`,
+    paddingTop: HEADING_FONT_SIZE_MOBILE * 0.6,
+    paddingBottom: HEADING_FONT_SIZE_MOBILE * 0.6,
+    [mediaQueries.mUp]: {
+      paddingTop: HEADING_FONT_SIZE_DESKTOP * 0.6,
+      paddingBottom: HEADING_FONT_SIZE_DESKTOP * 0.6
+    },
+    '&:last-child': css({
+      borderBottom: `1px solid ${colors.divider}`
+    })
+  }),
+  heading: css({
+    fontSize: HEADING_FONT_SIZE_MOBILE,
+    [mediaQueries.mUp]: {
+      fontSize: HEADING_FONT_SIZE_DESKTOP
+    }
+  }),
+  content: css({
+    marginBottom: 20,
+    /* paddingLeft: 0,
+    [mediaQueries.mUp]: {
+      paddingLeft: 55
+    } */
+    '&:last-child': css({
+      paddingBottom: HEADING_FONT_SIZE_MOBILE * 0.6,
+      [mediaQueries.mUp]: {
+        paddingBottom: HEADING_FONT_SIZE_DESKTOP * 0.6
+      },
+      borderBottom: `1px solid ${colors.divider}`
+    })
+  }),
+  contentFooter: css({
+    marginTop: 20
+  }),
+  doneIcon: css({
+    height: HEADING_FONT_SIZE_MOBILE * 1.1,
+    width: HEADING_FONT_SIZE_MOBILE * 1.1,
+    alignSelf: 'flex-end',
+    color: colors.primary,
+    [mediaQueries.mUp]: {
+      height: HEADING_FONT_SIZE_DESKTOP * 1.1,
+      width: HEADING_FONT_SIZE_DESKTOP * 1.1
+    }
+  })
+}
+
+export default (props) => {
+  const onExpand = e => {
+    e.preventDefault()
+    props.onExpand.apply(null, [props])
+  }
+
+  const onContinue = e => {
+    e.preventDefault()
+    props.onContinue.apply(null, [props])
+  }
+
+  const {
+    heading,
+    isLoading,
+    isTicked,
+    isVisited,
+    isExpanded,
+    error,
+    children,
+    forwardedRef,
+    showContinue = true
+  } = props
+
+  return (
+    <Fragment>
+      <div ref={forwardedRef} {...styles.section} onClick={onExpand}>
```

Comment:
Could have used [`React.forwardRef`](https://reactjs.org/docs/forwarding-refs.html) instead. But fine like this too.